evojax/algo/__init__.py

```diff
@@ -15,6 +15,7 @@
 from .base import NEAlgorithm
 from .cma_wrapper import CMA
 from .pgpe import PGPE
+from .cmaes import CMAES
```

Comment:
There can be multiple CMA-ES implementations in EvoJAX, we are considering to use the naming rule: `{algo_name}_{src_name}`.
For example, we will change ours to something like `CMAES_OriginalCPU`, and yours may be `CMAES_CyberAgent`.

Reply:
OK, I will update so.